front/app/components/UI/FullscreenModal/index.tsx

```diff
@@ -208,16 +210,18 @@ class FullscreenModal extends PureComponent<Props, State> {
       animateInOut,
       navbarRef,
       mobileNavbarRef,
+      className,
     } = this.props;
     const shards = compact([navbarRef, mobileNavbarRef]);
-    const modalPortalElement = document?.getElementById('modal-portal');
+    const modalPortalElement =
+      this.props.modalPortalElement || document?.getElementById('modal-portal');
```

Comment:
Added this because we're using a specific portal element for the mobile nav. There was an issue with tab order, because the expanded menu was higher in the DOM than the initial, shorter mobile navbar. So I had to put a portal element below the navbar in App/index.tsx.